star/private/pgstar_rti.f90

```diff
@@ -93,7 +93,7 @@ subroutine do_rti_Plot(s, id, device_id, &
 
          integer :: ix,k
          real :: xleft,xright,now
-         real :: dxmin=-1.d0
+         real, parameter :: dxmin = -1.d0
```

Comment:
This isn't a `parameter` in the previous file. Is that deliberate? This looks like it might be something that should be defined to a computed positive value later.